src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -349,6 +349,14 @@ protected override void OnHostStarted()
             _secretManager.PurgeOldFiles(Instance.ScriptConfig.RootScriptPath, Instance.TraceWriter);
         }
 
+        protected override void OnTimeout()
+        {
+            base.OnTimeout();
+            Stop();
```

Comment:
This shutdownTimeout setting needs some more investigation -- I'll handle that outside of this change.

---
In reply to: [76087184](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r76087184) [](ancestors = 76087184,76060160,75968751)